sdk/core/Azure.Core.TestFramework/tests/SessionRecords/RecordedTestBaseTests/WaitForCompletionErrorsArePropagated.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The Azure.Core changes need to be removed.

Reply:
removed

Reply:
@jsquire I removed Azure.Core changes, but now I'm getting "Generated code is not up to date." error. 

running 'eng\scripts\CodeChecks.ps1 -ServiceDirectory hybridcompute' succeeds.

Reply:
updated